### ISSUE TITLE
Is the post processing identical to the original paper?

### ISSUE THREAD

**Issue by shreyasrajesh** [ildoonet/tf-pose-estimation#295]
Hi,
I would be really grateful if someone could clarify whether post processing steps and methods used are identical to the original code provided in Matlab/C++ by the authors (https://github.com/ZheC/Realtime_Multi-Person_Pose_Estimation and https://github.com/CMU-Perceptual-Computing-Lab/openpose). It would also be wonderful if you could point out any of the major changes. Thanks.

**Comment by chrisskab** [comment]
Did you figure it out ?

**Comment by shreyasrajesh** [comment]
Yes, I compared the two completely, the only difference is in one of the steps of the postprocessing where 3x3 maxpooling is done to estabilish the peaks in the heatmaps in this repository whereas the original authors have done an array comparison with the left, right, top, bottom elements at each position to estabilish the peaks from the heatmaps. I haven't noticed any other algorithmic difference apart from this. I believe this should have extremely minimal effect on the eventual results except for really crowded cases (i.e. peaks exist at neighbouring diagonal pixels)
